CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+- TTID/TTFD Improvements ([#2866](https://github.com/getsentry/sentry-dart/pull/2866))
+```dart
+final sentryDisplay = Sentry.currentDisplay();
+// Do some work....
+sentryDisplay?.reportFullyDisplayed()
+```
```

Comment:
let's be a bit more specific on how to the new API works and also pls describe the widget